css/style.css

```diff
@@ -288,5 +288,30 @@ footer p {
   font-weight: 600;
 }
 
+.hamburger {
+  display: none;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
the rubric calls for modifying the font size of the page within your media query. you could do this by redefining the body font size: e.g. 

```
body {
 font-size: 18px;
}
```